test/utils_spec.js

```diff
@@ -843,6 +843,22 @@ describe("utils", function () {
           });
           expect(url).to.eql("http://res.cloudinary.com/test123/image/upload/if_w_lt_200,c_fill,h_120,w_80/sample");
         });
+        describe('conditional duration video', function () {
```

Comment:
@strausr , we need to also test the scenario of:
if: "initialDuration > 30" and assert that it's translated to "idu_gt_30"
Also check "duration" and "initial_duration"